src/fullscreen/fullscreen-controller.js

```diff
@@ -100,7 +100,7 @@ class FullscreenController {
         fullScreenElement = this._player.getView();
       }
       if (this._player.env.os.name === 'iOS') {
-        if (playbackConfig.inBrowserFullscreen && playbackConfig.playsinline) {
+        if ((playbackConfig.inBrowserFullscreen && playbackConfig.playsinline) || this._player._engineType === 'youtube') {
```

Comment:
@Tzipi-kaltura  can we use constant instead of hard coded?

Reply:
@giladna , done.